odl/trafos/wavelet.py

```diff
@@ -70,20 +61,34 @@ def coeff_size_list(shape, nscales, wbasis, mode):
     mode : `str`
         Signal extention mode. Possible extension modes are
 
-        'zpd': zero-padding -- signal is extended by adding zero samples
+        'zero': zero-padding -- signal is extended by adding zero samples
 
-        'cpd': constant padding -- border values are replicated
+        'constant': constant padding -- border values are replicated
 
-        'sym': symmetric padding -- signal extension by mirroring samples
+        'symmetric': symmetric padding -- signal extension by mirroring samples
 
-        'ppd': periodic padding -- signal is trated as a periodic one
+        'periodic': periodic padding -- signal is trated as a periodic one
 
-        'sp1': smooth padding -- signal is extended according to the
+        'smooth': smooth padding -- signal is extended according to the
         first derivatives calculated on the edges (straight line)
 
-        'per': periodization -- like periodic-padding but gives the
+        'periodization': periodization -- like periodic-padding but gives the
```

Comment:
Great change, much more readable!

A small comment: since this changes the API (and the function is public), such a change should have "API:" in the commit message. Maybe you did that. If not: if you happen to have this isolated in one commit, change the commit message with `git rebase -i`, otherwise leave it and do it next time :-)